docs/source/demos/lead_scoring.ipynb

```diff
@@ -1,6 +1,7 @@
 {
  "cells": [
   {
+   "attachments": {},
```

Comment:
Any idea why these got added here and removed from timeseries.ipynb?

Reply:
Ah, I think it's thrown in when you open the ipynb in VSCode. I thought I manually deleted some of these before I pushed. I'll take them out!